scripts/obtain/dotnet-install.ps1

```diff
@@ -140,6 +140,17 @@ function GetHTTPResponse([Uri] $Uri)
     try {
         # HttpClient is used vs Invoke-WebRequest in order to support Nano Server which doesn't support the Invoke-WebRequest cmdlet.
         Load-Assembly -Assembly System.Net.Http
+ 
+        if(-not $ProxyAddress)
```

Comment:
@omajid  yes I agree, that's the standard way to do it on Linux platforms so no need to do anything on the .sh side